src/app/App.js

```diff
@@ -18,129 +20,97 @@ import { ModeRanking } from './components/ModeRanking';
+  //   component: GameMode(),
+  //   inside: mainContainer,
+  //   withClasses: 'mainContainer__gameMode',
+  // });
+
   // const modeRules = render({
   //   //component: ModeRules('You have one minute (1m) to answer as many questions as possible. During the game on each question you need to select who from Star Wars is showed on the left (Jar Jar Binks right now) from available options'),
   //   component: ModeRules(),
   //   inside: mainContainer,
   //   withClasses: 'mainContainer__modeRules',
   // });
 
-  const topScores = [
-    {
-      user: 'user1',
-      score: 15,
-      maxScore: 30,
-    },
-    {
-      user: 'user2',
-      score: 18,
-      maxScore: 40,
-    },
-    {
-      user: 'user3',
-      score: 15,
-      maxScore: 25,
-    },
-  ];
-
-  const modeRanking = render({
-    component: ModeRanking(topScores),
-    inside: mainContainer,
-  });
-
-  const hallOfFameButton = render({
-    component: Button({
-      id: 'hallOfFameButton',
-      btnText: 'HallOfFame',
-      classList: ['mainContainer__hallOfFameButton'],
-      onClickFn: undefined,
-      icon: 'fame',
-    }),
-    inside: mainContainer,
-  });
+  // const topScores = [
+  //   {
+  //     user: 'user1',
+  //     score: 15,
+  //     maxScore: 30,
+  //   },
+  //   {
+  //     user: 'user2',
+  //     score: 18,
+  //     maxScore: 40,
+  //   },
+  //   {
+  //     user: 'user3',
+  //     score: 15,
+  //     maxScore: 25,
+  //   },
+  // ];
+
+  // const modeRanking = render({
+  //   component: ModeRanking(topScores),
+  //   inside: mainContainer,
+  // });
 
-  const playTheGameButton = render({
-    component: Button({
-      id: 'playTheGameButton',
-      btnText: 'play the game',
-      classList: ['mainContainer__playTheGameButton'],
-      onClickFn: startGame,
-    }),
-    inside: mainContainer,
-  });
-  playTheGameButton.setSpecial();
-  const questionAnswers = render({
-    component: QuestionAnswers(
-      ['Luke Skywalker', 'Jar Jar Binks', 'Padme Amidala', 'Darth Vader'],
-      'Darth Vader',
-      onAnswerChosen,
-    ),
-    //component: QuestionAnswers(answers, correctAnswer, onAnswerChosen),
-    inside: mainContainer,
-    withClasses: 'mainContainer__answersWrapper',
-  });
+  // playTheGameButton.setSpecial();
+  // const questionAnswers = render({
+  //   component: QuestionAnswers(
+  //     ['Luke Skywalker', 'Jar Jar Binks', 'Padme Amidala', 'Darth Vader'],
+  //     'Darth Vader',
+  //     onAnswerChosen,
+  //   ),
+  //   //component: QuestionAnswers(answers, correctAnswer, onAnswerChosen),
+  //   inside: mainContainer,
+  //   withClasses: 'mainContainer__answersWrapper',
+  // });
 
   // Modal window
-  const modalWindow = ModalWindow(app);
+  // const modalWindow = ModalWindow(app);
   // Modal window methods
   // modalWindow.show(Here_some_content_as_HTMLElement);
   // modalWindow.close  to close the window and remove the content
 
-  /* MODAL WINDOW CONTENT TO REMOVE LATER!
```

Comment:
Potrzebne to zakomentowanie :) ? Jak ten kod się na coś przyda to macie Gita. Po prostu sprawdzicie historię.

Reply:
To nie nasze, to chyba Pawła, zaciągnięta w pullRequest (od Pawla)